hazelcast/src/main/java/com/hazelcast/util/ExceptionUtil.java

```diff
@@ -28,7 +28,10 @@
  */
 public final class ExceptionUtil {
 
-    private static final String EXCEPTION_SEPARATOR = "------ End remote and begin local stack-trace ------";
+    /**
+     * Separator for local/remote stacktrace.
+     */
+    public static final String EXCEPTION_SEPARATOR = "------ End remote and begin local stack-trace ------";
```

Comment:
a nitpick: I always have to think hard to tell what part of the stacktrace is the local part and what is the remote part. Perhaps the separator could indicate this?